### ISSUE TITLE
FYPO:0004021	long polar microtubules

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3026]
Comment from Meredith:

The definition of this term is not right
I suggest an update to FYPO to distinguish between mitotic interpolar microtubules and mitotic polar microtubules. The current term describing 'polar microtubules' - meaning microtubules that form the bipolar spindle - should be called 'interpolar microtubules.' The term 'polar microtubules' should be redefined to mean spindle microtubules that come from each pole but do *not* overlap in the middle.

(I never knew this)

**Comment by ValWood** [comment]
"We use the term polar MTs for single or bundled nuclear MTs emanating from an SPB and the term interpolar MTs for the shaft of interdigitating polar MTs that
runs between the poles; in some studies, the latter have been called the central or core spindle. (Note that in anaphase, cytoplasmic as- tral MTs develop; these have not been measured in our study.) "

**Comment by mah11** [comment]
oh boy, here we go ...

At present, FYPO follows GO for the definition of "polar microtubule" (GO:0005827), and deviating would be Bad Ontology Practice™. So this ought to be updated in GO CC first and then I'll make FYPO consistent with it.

The GO:0005827 definition goes back a long time, and I suspect no one knew then that there were spindle MTs that aren't astral and neither connect to a kinetochore nor interdigitate with MTs from the opposite pole.

Furthermore, I would *very strongly* discourage GO from changing the meaning of "polar microtubule" from "ones that overlap" to "ones that explicitly don't overlap", and I really wish Meredith's lab hadn't done so. The current def is so long-established that GO cites a textbook for it, and the term has been used with that meaning extensively for a decade and a half.


Instead, in GO, "polar microtubule" could be broadened to be a parent of both overlapping and non-overlapping MTs that extend from a pole along the long axis of the spindle. They'll need a name for the non-overlapping MTs that Meredith calls "polar MTs" ... hope I can come up with something nicer than "non-overlapping polar microtubule" :P I like using "interpolar microtubule" for the MTs that overlap (old meaning of "polar MTs").

i.e. suggest to GO

    polar microtubule GO:new1 (or GO:0005827?) - any MT that originates at one spindle pole and extends along the long axis of the spindle
    - [i] interpolar microtubule - GO:0005827 (or GO:new1?) - any MT that originates at one spindle pole, extends along the long axis of the spindle, and overlaps at the spindle midzone with microtubules originating at the opposite pole
    - [i] NEED NAME GO:new2 - any MT that originates at one spindle pole and extends along the long axis of the spindle, but does not overlap with microtubules originating at the opposite pole

and the name of GO:new2 should *not* be "polar microtubule"

**Comment by ValWood** [comment]
Hmmph. Will e-mail Meredith for clarification tomorrow... Too tired now for this one ;)

**Comment by mah11** [comment]
did that paper look as complicated to you as it did to me?

**Comment by ValWood** [comment]
Yes! Meredith did a brilliant job...she came to the curation session in Banff and then she was away.
I could not have done it  as this well myself without taking 2 days I'm sure! its very cool stuff though.
